packages/playground/website/src/components/playground-viewport/index.tsx

```diff
@@ -21,52 +28,110 @@ interface PlaygroundViewportProps {
 	className?: string;
 }
 
-export const PlaygroundViewport = React.forwardRef<
-	HTMLIFrameElement,
-	PlaygroundViewportProps
->(
-	(
-		{ displayMode = 'browser', toolbarButtons, hideToolbar, className },
-		ref
-	) => {
-		const { playground, currentUrl: url } = usePlaygroundContext();
+export const PlaygroundViewport = ({
+	displayMode = 'browser-full-screen',
+	toolbarButtons,
+	hideToolbar,
+	className,
+}: PlaygroundViewportProps) => {
+	const clientInfo = useAppSelector(getActiveClient);
+	const playground = clientInfo?.client;
+	const url = clientInfo?.url;
 
-		if (displayMode === 'seamless') {
-			// No need to boot the playground if seamless.
-			return <JustViewport iframeRef={ref} />;
-		}
-		return (
-			<BrowserChrome
-				showAddressBar={!!playground}
-				url={url}
-				toolbarButtons={toolbarButtons}
-				onUrlChange={(url) => playground?.goTo(url)}
-				hideToolbar={hideToolbar}
-				className={className}
-			>
-				<JustViewport iframeRef={ref} />
-			</BrowserChrome>
-		);
+	if (displayMode === 'seamless') {
+		// No need to boot the playground if seamless.
```

Comment:
I think this may be a stale comment since `<JustViewport />` currently calls bootPlayground().

Reply:
yup